flat_mutation_reader_v2.hh

```diff
@@ -444,6 +447,8 @@ private:
     void do_upgrade_schema(const schema_ptr&);
     static void on_close_error(std::unique_ptr<impl>, std::exception_ptr ep) noexcept;
 public:
+    operator flat_mutation_reader() &&;
```

Comment:
This is declared, but not defined?

Reply:
Sorry, this shouldn't land in this commit. Now I removed this line.